api/src/score/dto/score.dto.ts

```diff
@@ -160,5 +160,5 @@ export class ScoreDto {
   finalScore: number;
 
   @IsEnum(CompletionStatus)
-  completionStatus: CompletionStatus;
+  status: CompletionStatus;
```

Comment:
this is pulling the enum from `'../../common/enums'`

Reply:
Yes, it comes as a string not entity first. Then I make a call to get it from database based on name